packages/twenty-front/src/modules/object-record/record-inline-cell/components/RecordInlineCellContainer.tsx

```diff
@@ -41,11 +41,29 @@ const StyledLabelAndIconContainer = styled.div`
   height: 24px;
 `;
 
-const StyledValueContainer = styled.div`
+const StyledValueContainer = styled.div<{ readonly: boolean }>`
+  cursor: ${({ readonly }) => (readonly ? 'default' : 'pointer')};
   display: flex;
   flex-grow: 1;
   min-width: 0;
   position: relative;
+
+  &:hover {
```

Comment:
I would say the right direction regarding performances is to add a cell-readonly class in the html, instead of passing a props to the Styled, but this can be revisited later